app/controllers/steps/client/retry_benefit_check_controller.rb

```diff
@@ -0,0 +1,17 @@
+module Steps
+  module Client
+    class RetryBenefitCheckController < Steps::ClientStepController
```

Comment:
Check the styling on any of the pages with a green and a grey button next to each other and we can replicate it here to look the same. I think it may just need a wrapper `<div class="govuk-button-group">`